docs/classes.list.js

```diff
@@ -113,51 +132,43 @@ export const borderColor = [
   'border-current',
   'border-inverted',
   'border-transparent',
-  's-border-default',
+  's-border',
   's-border-hover',
   's-border-active',
-  's-border-active-hover',
```

Comment:
We have removed `s-border-xxx-active-hover` (You can check [here](https://docs.google.com/spreadsheets/d/1Q-Tr_dwJVfxgh3527IFjXQKqtnPPM42QeIr-M6q-zu8/edit#gid=888578831) and it is not in semantic.yml anymore)